js/genkit/src/genkit-beta.ts

```diff
@@ -16,24 +16,28 @@
 
 import {
   defineInterrupt,
+  generateOperation,
+  GenerateOptions,
+  GenerationCommonConfigSchema,
   isExecutablePrompt,
   type ExecutablePrompt,
   type InterruptConfig,
   type ToolAction,
 } from '@genkit-ai/ai';
 import type { Chat, ChatOptions } from '@genkit-ai/ai/chat';
 import { defineFormat } from '@genkit-ai/ai/formats';
+import { ModelOperation } from '@genkit-ai/ai/model';
 import {
+  getCurrentSession,
   Session,
   SessionError,
-  getCurrentSession,
   type SessionData,
   type SessionOptions,
 } from '@genkit-ai/ai/session';
 import type { z } from '@genkit-ai/core';
 import { v4 as uuidv4 } from 'uuid';
-import type { Formatter } from './formats';
-import { Genkit, type GenkitOptions } from './genkit';
+import type { Formatter } from './formats.js';
```

Comment:
Why add the '.js'? for './formats' and './genkit'?

Reply:
It's a requirement for ESM. When importing a file directly it must end in .js....